src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -122,20 +116,23 @@ public function raCandidateSearchAction(Request $request)
 
         $logger->notice(sprintf('Searching for RaCandidates within institution "%s"', $institution));
 
+        $service = $this->getRaCandidateService();
+
         $command                   = new SearchRaCandidatesCommand();
         $command->actorId          = $this->getUser()->id;
         $command->actorInstitution = $institution;
         $command->pageNumber       = (int) $request->get('p', 1);
         $command->orderBy          = $request->get('orderBy');
         $command->orderDirection   = $request->get('orderDirection');
 
+        $raCandidateList = $service->search($command);
+
         // The options that will populate the institution filter choice list.
-        $command->institutionFilterOptions = $institutionFilterOptions;
+        $command->institutionFilterOptions = $raCandidateList->getFilterOption('institution');
```

Comment:
Now that you are no longer using the previously declared `$institutionFilterOptions`, you can remove it [here on line 113](https://github.com/OpenConext/Stepup-RA/pull/211/files#diff-400979a26c939611078fbafdbaed6154R113)

And with that, you can also clean up the [getAvailableInstitutionsFor](https://github.com/OpenConext/Stepup-RA/blob/4e3823bd883d4f2af252ea5e313fe50e5be11aaf/src/Surfnet/StepupRa/RaBundle/Service/InstitutionConfigurationOptionsService.php#L75) method if I grepped correctly (please verify this!)

Reply:
I've fixed the unused methods